src/execution/operator/csv_scanner/sniffer/header_detection.cpp

```diff
@@ -149,6 +148,24 @@ bool CSVSniffer::DetectHeaderWithSetColumn(ClientContext &context, vector<Header
 	return has_header;
 }
 
+bool EmptyHeader(const string &col_name, bool is_null, bool normalize) {
+	if (col_name.empty() || is_null) {
+		return true;
+	}
+	if (normalize) {
+		// normalize has special logic to trim white spaces and generate names
+		return false;
+	}
+	// check if it's all white spaces
+	for (auto &c : col_name) {
+		if (c != ' ') {
```

Comment:
Can we use `StringUtil::CharacterIsSpace` here so that we also capture other types of spaces (e.g. tabs)?